doc/user_guide/nemo.rst

```diff
@@ -1,7 +1,7 @@
 .. -----------------------------------------------------------------------------
 .. BSD 3-Clause License
 ..
-.. Copyright (c) 2018-2019, Science and Technology Facilities Council.
+.. Copyright (c) 2018-2020, Science and Technology Facilities Council.
```

Comment:
21 now please.

Reply:
git log shows the last update to this file being Dec 2020. Updated to 2021 anyway.